src/commands/build.ts

```diff
@@ -19,16 +23,33 @@ export default class Build extends BaseBuildCommand {
 
   static flags = {
     'all-ms': Flags.boolean({
-      description: 'Builds all the bundle microservices'
+      description: 'Build all the bundle microservices'
+    }),
+    'all-mfe': Flags.boolean({
+      description: 'Build all the bundle micro frontends'
     })
   }
 
   public async run(): Promise<void> {
     BundleService.verifyBundleInitialized(process.cwd())
     const { args, flags } = await this.parse(Build)
-
     const componentService = new ComponentService()
-    if (flags['all-ms']) {
+
+    if (Object.keys(flags).length > 1) {
+      this.error(`Build failed, please use only one flag`, { exit: 1 })
+    }
+
+    if (Object.keys(flags).length > 0 && args.name !== undefined) {
+      this.error(
+        `Build failed, please use one flag or write the component name as argument`,
+        { exit: 1 }
+      )
+    }
+
+    if (flags['all-mfe']) {
+      CliUx.ux.action.start(`Building all microfrontends`)
+      await this.buildAllComponents(Phase.Build, ComponentType.MICROFRONTEND)
```

Comment:
I don't think we should use the spinner together with the progress bar. It causes the progress bar to split during the update, like this:
```
progress [----------------------------------------] 0% | ETA: 0s | 0/2
progress [========================================] 100% | ETA: 0s | 2/2
```

Reply:
spinner removed